src/components/Counter/Index.tsx

```diff
@@ -1,30 +1,48 @@
 import { FiMinus, FiPlus } from 'solid-icons/fi'
 import { Component } from 'solid-js'
 import IconButton from '@components/Buttons/IconButton/Index'
-
+import Slider from '@components/Slider/Index'
 export interface IProps {
     onClickIncrease: () => void
     onClickDecrease: () => void
-    value: string
+    onInput: (value: string) => void
+    value: number
+    label: string
+    min: string
+    max: string
```

Comment:
I'm still not sure about this part, shouldn't these be numbers after all? We're getting them as number from the user eitherway so we'd avoid converting them back and forth  in CameraPanel and here